spec/factories/products.rb

```diff
@@ -15,5 +15,46 @@
       main_display_image { Rack::Test::UploadedFile.new(Rails.root.join('app/assets/images/about-page-image.png'), 'image/jpg') }
       main_display_image_alt_text { "sample image" }
     end
+
+    trait :with_multimedia do
+      after(:create) do |product|
+        # Video
+        PracticeMultimedium.create(
+          link_url: "https://www.youtube.com/watch?v=-oazAtTm-lk",
+          name: "Dr. Jeffrey Heckman speaks about the collaborative origins of FLOW3 and how it
+          uses three custom-designed features to address issues with artificial limb authorizations.)",
+          resource_type: "video",
+          innovable: product
+        )
+        # Image
+        PracticeMultimedium.create(
+          link_url: "https://www.youtube.com/watch?v=-oazAtTm-lk",
```

Comment:
it looks like the existing `practice_multimedia` in the db with a `resource_type` of "image" have nil values for the `:link_url` attribute

Reply:
Ooh good catch, thanks!